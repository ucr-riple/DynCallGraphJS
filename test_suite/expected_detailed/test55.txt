{
    'global (test55.js:1:1:11:5)': [
        'anon (test55.js:1:2:11:2)'
    ],
    'anon (test55.js:9:3:9:9)': [
        'foo (test55.js:4:3:6:4)'
    ],
    'anon (test55.js:10:3:10:9)': [
        'foo (test55.js:4:3:6:4)'
    ]
}