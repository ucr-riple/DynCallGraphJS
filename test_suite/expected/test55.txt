{ '(test55.js:1:1:11:5)':
   [ '(test55.js:1:2:11:2)' ],
  '(test55.js:9:3:9:9)':
   [ '(test55.js:4:3:6:4)' ],
  '(test55.js:10:3:10:9)':
   [ '(test55.js:4:3:6:4)' ] }